Services/Mail/PRIVACY.md

```diff
@@ -0,0 +1,80 @@
+# Mail Service Privacy
+
+This documentation does not warrant completeness or correctness. Please report any
+missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de)
+or contribute a fix via [Pull Request](docs/development/contributing.md#pull-request-to-the-repositories).
+
+## General Information
+
+- The `Mail Service` can be disabled in the global administration or restricted to user accounts assigned
```

Comment:
- Please consider moving this information to "Data being presented". 
- Please consider being more specific: An account with "Edit Settings" permissions for Administration > Communication > Mail > Settings can enable the External E-Mail globally.
- An account with "Edit Settings" permissions for Administration > User and Roles > Roles > RoleTitle > Default Permissions > Administration Permission > Mail can remove the permission "Internal Mail: User can use internal mail system" for all users of that role.